src/Features/LanguageServer/ProtocolUnitTests/DocumentChanges/DocumentChangesTests.LinkedDocuments.cs

```diff
@@ -107,40 +104,11 @@ void M()
             Assert.Empty(testLspServer.GetQueueAccessor().GetTrackedTexts());
         }
 
-        private static async Task<Solution> GetLSPSolution(TestLspServer testLspServer, Uri uri)
-        {
-            var result = await testLspServer.ExecuteRequestAsync<Uri, Solution>(nameof(GetLSPSolutionHandler), uri, new ClientCapabilities(), null, CancellationToken.None);
```

Comment:
solutions (esp from the testworkspace) are not easily serializable.  Easier to just verify that the manager returns the correct solution